File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/erf/ErfPlot.java
package net.fwitz.math.main.binary.complex.functions.erf;

import net.fwitz.math.binary.complex.functions.Erf;
import net.fwitz.math.plot.binary.complex.color.DomainColoringSmooth;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class ErfPlot {
    public static void main(String[] args) {
        new ComplexFunctionPlot("erf(z)")
                .computeFn(Erf::erf)
                .domainX(-5, 5)
                .domainY(-5, 5)
                .colorFn(new DomainColoringSmooth())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/erf/ErfPlot.kt
package net.fwitz.math.main.binary.complex.functions.erf

import net.fwitz.math.binary.complex.functions.Erf
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot
import net.fwitz.math.plot.binary.complex.color.DomainColoringSmooth

object ErfPlot {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("erf(z)")
        .computeFn(Erf::erf)
        .domainX(-5.0, 5.0)
        .domainY(-5.0, 5.0)
        .colorFn(DomainColoringSmooth)
        .render()
}